### PR TITLE
Adding helm dependency update command

### PR DESCRIPTION
**Pull request by shigupt202**
Adding helm dependency update command before helm template command

### CODE REVIEW

**Comment by shigupt202** [comment]
Fixes #4

**Comment by s-tuli** [comment]
Thanks @shigupt202 for working on this.

File: src/run.ts [hunk 1]
```diff
@@ -29,14 +29,23 @@ abstract class RenderEngine {
...
+
+        const chartPath = core.getInput('helmChart', {required : true});
+        const options = {
             silent: true
-         } as ExecOptions;
+        } as ExecOptions;
+
+        var dependencyArgs = this.getDependencyArgs(chartPath);
+        
+        core.debug("Running helm dependency update command..");
```
Comment (ds-ms) [inline src/run.ts]: This can be console.log

File: src/run.ts [hunk 2]
```diff
@@ -29,14 +29,23 @@ abstract class RenderEngine {
...
+        await utilities.execCommand(helmPath, dependencyArgs, options);
+
+        core.debug("Creating the template argument string..");
+        var args = this.getTemplateArgs(chartPath)
         
         core.debug("Running helm template command..");
```
Comment (ds-ms) [inline src/run.ts]: This too.

Comment (shigupt202) [reply src/run.ts]: This was core.debug already. Should I still update it?

File: src/run.ts (hunk 1 above)
Comment (shigupt202) [reply src/run.ts]: Done

**Pull request merged**